openapi/components/schemas/Cashier/CashierStrategyCustomAmount.yaml

```
@@ -0,0 +1,30 @@
+type: object
+required:
+  - minimum
+  - multipleOf
+  - maximum
+properties:
+  minimum:
+    description: Minimum custom amount.
+    type: number
+    format: double
+    minimum: 0
+    exclusiveMinimum: true
+  multipleOf:
```

May be "**step**" is a better name here? 
```suggestion
  step:

---

suggestion: how about `multiplier` instead?

---

I agree with @lexerom . `multiplier` can be misleading since, without reading description, I would assume the progression will follow as:
```php
$minimum = 2.4;
$multiplier = 2;
$result = [2.4, 4.8, 9.6 ...];
```